app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+  after_save :increase_posts_counter
+
+  def increase_posts_counter
+    author.increment!(:posts_counter)
+  end
```

Comment:
- [OPTIONANL] It would be nice to make the `update_post_counter ` method private since it serves only inside this class.

Reply:
Hi, thanks for the feedback, I will do it